client/components/button/button.html

```diff
@@ -0,0 +1 @@
+<button className="unique-button-class">Click me</button>
```

Comment:
`class` :) 
you made be to debug for a while xD

Reply:
ahaha 🥲 What React did to me...

Reply:
```suggestion
<button class="unique-button-class">Click me</button>
```